integration_tests/debug.sh

```
@@ -0,0 +1,51 @@
+cmdstring=""
+
+if [ -n "$1" ]; then
+ cmdstring+=" -protocol $1 "
+fi
+
+if [ -n "$2" ]; then
+ cmdstring+=" -template $2 "
+fi
+
+# Parse any extra args that are directly passed to nuclei
+if [ -n $debugArgs ]; then
```

Shellcheck (0.11.0)

[error] -n doesn't work with unquoted arguments. Quote or use [[ ]].

(SC2070)